types/events.go

```diff
@@ -113,6 +115,12 @@ type IntelRdt struct {
 
 	// The memory bandwidth schema in 'container_id' group
 	MemBwSchema string `json:"mem_bw_schema,omitempty"`
+
+	// The memory bandwidth monitoring statistics from NUMA nodes in 'container_id' group
+	MBMStats *[]intelrdt.MBMNumaNodeStats `json:"mbm_statistics,omitempty"`
```

Comment:
maybe: `json:"mbm_stats,omitempty"`

Reply:
good idea

Reply:
@Creatone ^^^

Reply:
@kolyshkin this comment is outdated. String was already changed long time ago.